src/main/java/com/google/api/codegen/discovery/CommonDiscoveryProvider.java

```diff
@@ -45,12 +45,8 @@ public CommonDiscoveryProvider(
 
     Api api = context.getApi();
     String outputRoot =
-        "autogenerated/"
-            + api.getName()
-            + "/"
-            + api.getVersion()
-            + "/"
-            + context.getService().getDocumentation().getOverview();
+        "autogenerated/" + api.getName() + "/" + api.getVersion() + "/" + context.getApiRevision();
```

Comment:
I think it has already been run; test wouldn't pass otherwise. The line got shorter, GJF might pull it back to the same line.